lib/image.js

```diff
@@ -62,7 +68,13 @@ Object.defineProperty(Image.prototype, 'src', {
       this._originalSource = undefined
     }
   },
-
+  onHttpError: err => {
```

Comment:
`onHttpError` should not be defined here. This object is a temporary object that is passed to `Object.defineProperty`. You can use something like

```js
Image.prototype.onHttpError = function(err){
  if (typeof this.onerror === 'function') {
    this.onerror(err)
  } else {
    throw err
  }
};
```

Also, it should not be an arrow function if you want to access `this`